alica_engine/src/engine/BasicPlan.cpp

```diff
@@ -28,12 +27,6 @@ BasicPlan::BasicPlan(PlanContext& context)
 
 void BasicPlan::doInit()
 {
-    // TODO Cleanup: do this after moving trace context setting to RunningPlan
-    /*if (_isMasterPlan) {
```

Comment:
We still need this right?

Reply:
I don't really get this tbh

1) This is commented out so it isn't needed at the moment, and it isn't clear exactly when it would be needed.  This is referring to a refactor idea that has no wrike task
2) Children trace should show up (with no parent) even if the master plan trace has not terminated

Reply:
my bad, I didn't see that it was already commented